### ISSUE TITLE
GimmeWidget is firing off 7 requests for a single data point?

### ISSUE THREAD

**Issue by brucecrevensten** [ua-snap/ardac-explorer#118]
STR:

Env: API = development.earthmaps.io / Rasdaman = zeus 

 - http://localhost:3000/item/permafrost-magt
 - open network tab of dev console
 - pick Fairbanks as a place

I'm seeing 7 HTTP requests fired off for the same target data.  This doesn't seem right!

**Comment by cstephen** [comment]
It looks like the number of identical API HTTP requests correspond to the number of charts that are displayed on the ARDAC item. Since the permafrost-magt item displays 7 charts (one chart for each MAGT depth), it fires off 7 requests. Similarly, the permafrost base & top item fires off 2 identical HTTP requests since it displays two charts.

I haven't dug into this very deep, but here's the watch function that fires off the permafrost API HTTP request:

https://github.com/ua-snap/ardac-explorer/blob/b30b110460beee6902d53ba26903204144282a9f/components/PermafrostChart.vue#L222-L226

And I suspect this line is the culprit:

https://github.com/ua-snap/ardac-explorer/blob/b30b110460beee6902d53ba26903204144282a9f/components/PermafrostChart.vue#L18

In other words, it appears that each time `PermafrostChart.vue` is mounted, it sets a new `latLng` computed property for that instance of `PermafrostChart.vue` and triggers another API fetch. There are probably several ways to fix this, but will almost certainly involve adding more intelligence to the watch function referenced above.
